index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JIC Portfolio</title>
+
+	<link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700|Poppins" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header class="Toolbar">
+    <a href="index.html" class="Mylogo Title2">JIC Software</a>
+    <div class="menu-pc">
+      <a class="item-menu-pc">Portfolio</a>
+      <a class="item-menu-pc">About</a>
+      <a class="item-menu-pc-group">
+      <a class="item-menu-pc">Contact</a>
+        <img class="item-menu-pc-icon" src="images/mail_icon.png" alt="email icon">
+      </a>
+    </div>
+    <nav class="menu-mobile">
+        <img class="burguer-menu" src="images/burguer_menu_icon.png" alt="burguer menu icon">
+    </nav>
+  </header>
+  <section class="main-frame">
+    <div class="content1">
+      <section class="Headline">
+        <h1 class="Primarytext Title1">Hey There.<br>I'm Jorge.</h1>
+        <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+        <p class="Descriptiontext Simpletext">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </section>
```

Comment:
Kindly increase the font size of the texts to match the template 👍 

Figma
![image](https://user-images.githubusercontent.com/78251927/227701529-ad7c93ab-2086-400f-968b-2d8bb639dfda.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/227701547-1a73adab-868a-430f-b3d8-dae6ec40de9c.png)

Reply:
Done. Thank you @abdulhamiid